src/base/bittorrent/torrenthandle.h

```diff
@@ -302,9 +302,10 @@ namespace BitTorrent
         int connectionsCount() const;
         int connectionsLimit() const;
         qlonglong nextAnnounce() const;
+        bool isAutoManaged() const;
```

Comment:
This property (auto_managed in terms of libtorrent) isn't supposed to be a part of qBittorrent TorrentHandle interface so please make its accessors private.

Reply:
Done.